src/ol/style/literal.js

```diff
@@ -29,7 +29,7 @@ export const SymbolType = {
 
 /**
  * @typedef {Object} LiteralSymbolStyle
- * @property {ExpressionValue|Array<ExpressionValue, ExpressionValue>} size Size, mandatory.
+ * @property {ExpressionValue|Array<ExpressionValue>} size Size, mandatory.
```

Comment:
I think the intention here was to either allow a single expression or an array of two expressions (and not a longer array).  I wonder if the right tuple syntax would have been `[ExpressionValue, ExpressionValue]`.

Reply:
Yes, i think you are correct. The size can be a value or a tuple. I can correct it in another PR.